app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments, foreign_key: 'author_id'
```

Comment:
- [ ] it looks like you might have copied and pasted this line form the user model, which is no problem, however, make sure to adjust it to the post model as well. please remove the foreign_key: 'author_id' in this line